### ISSUE TITLE
Failed to map stack to user code for Angular CLI with Jest schematic

### ISSUE THREAD

**Issue by smcenlly** [wallabyjs/public#2307]
![image](https://user-images.githubusercontent.com/13978154/71978766-6f89db00-321c-11ea-91dd-39801ee68278.png)

Clean Angular repo, Jest added with https://github.com/briebug/jest-schematic

```
{
  editorVersion: 'WebStorm 2019.3.2 Preview',
  pluginVersion: '1.0.146',
  editorType: 'IntelliJ',
  osVersion: 'win32 10.0.18362',
  nodeVersion: 'v12.11.1',
  coreVersion: '1.0.806',
  config: {
    diagnostics: {},
    testFramework: { version: 'jest@24.8.0', configurator: 'jest@24.8.0', reporter: 'jest@24.8.0', starter: 'jest@24.8.0', autoDetected: true },
    filesWithCoverageCalculated: [],
    micromatch: true,
    files: [
      { pattern: '\\\\node_modules\\\\', regexp: /\\node_modules\\/, ignore: true, trigger: true, load: true },
      {
        pattern: 'dist\\\\|build\\\\|coverage\\\\|\\\\\\.idea\\\\|\\\\\\.vscode\\\\|\\\\\\.git\\\\|\\\\\\.gitlab\\\\',
        regexp: /dist\\|build\\|coverage\\|\\\.idea\\|\\\.vscode\\|\\\.git\\|\\\.gitlab\\/,
        ignore: true,
        trigger: true,
        load: true
      },
      { pattern: 'src/**', ignore: false, trigger: true, load: true, order: 1 },
      { pattern: 'src/**/__tests__/**/*.[jt]s?(x)', ignore: true, trigger: true, load: true },
      { pattern: 'src/**/?(*.)+(spec|test).[tj]s?(x)', ignore: true, trigger: true, load: true }
    ],
    tests: [
      { pattern: '\\\\node_modules\\\\', regexp: /\\node_modules\\/, ignore: true, trigger: true, load: true, test: true },
      {
        pattern: 'dist\\\\|build\\\\|coverage\\\\|\\\\\\.idea\\\\|\\\\\\.vscode\\\\|\\\\\\.git\\\\|\\\\\\.gitlab\\\\',
        regexp: /dist\\|build\\|coverage\\|\\\.idea\\|\\\.vscode\\|\\\.git\\|\\\.gitlab\\/,
        ignore: true,
        trigger: true,
        load: true,
        test: true
      },
      { pattern: 'src/**/__tests__/**/*.[jt]s?(x)', ignore: false, trigger: true, load: true, test: true, order: 2 },
      { pattern: 'src/**/?(*.)+(spec|test).[tj]s?(x)', ignore: false, trigger: true, load: true, test: true, order: 3 }
    ],
    filesWithNoCoverageCalculated: [],
    runAllTestsInAffectedTestFile: false,
    compilers: {},
    preprocessors: {},
    maxConsoleMessagesPerTest: 100,
    autoConsoleLog: true,
    delays: { run: 0, edit: 100, update: 0 },
    workers: { initial: 0, regular: 0, recycle: false },
    teardown: undefined,
    hints: {
      ignoreCoverage: '__REGEXP /ignore coverage|istanbul ignore/',
      ignoreCoverageForFile: '__REGEXP /ignore file coverage/',
      commentAutoLog: '?',
      testFileSelection: { include: '__REGEXP /file\\.only/', exclude: '__REGEXP /file\\.skip/' }
    },
    automaticTestFileSelection: true,
    runSelectedTestsOnly: false,
    extensions: {},
    env: { type: 'node', params: {}, runner: 'F:\\app\\nodejs\\node.exe', viewportSize: { width: 800, height: 600 }, options: { width: 800, height: 600 }, bundle: true },
    reportUnhandledPromises: true,
    slowTestThreshold: 75,
    lowCoverageThreshold: 80,
    loose: true,
    configCode: 'auto.detect#-697624100'
  },
  packageJSON: {
    dependencies: {
      '@angular/animations': '~8.2.8',
      '@angular/common': '~8.2.8',
      '@angular/compiler': '~8.2.8',
      '@angular/core': '~8.2.8',
      '@angular/forms': '~8.2.8',
      '@angular/platform-browser': '~8.2.8',
      '@angular/platform-browser-dynamic': '~8.2.8',
      '@angular/router': '~8.2.8',
      '@briebug/jest-schematic': '^2.1.1',
      rxjs: '~6.4.0',
      tslib: '^1.10.0',
      'zone.js': '~0.9.1'
    },
    devDependencies: {
      '@angular-devkit/build-angular': '~0.803.6',
      '@angular/cli': '~8.3.6',
      '@angular/compiler-cli': '~8.2.8',
      '@angular/language-service': '~8.2.8',
      '@types/jest': '24.0.25',
      '@types/node': '~8.9.4',
      '@types/jasmine': '~3.3.8',
      '@types/jasminewd2': '~2.0.3',
      codelyzer: '^5.0.0',
      'jasmine-core': '~3.4.0',
      'jasmine-spec-reporter': '~4.2.1',
      jest: '24.9.0',
      'jest-preset-angular': '8.0.0',
      protractor: '~5.4.0',
      'ts-node': '~7.0.0',
      tslint: '~5.15.0',
      typescript: '~3.5.3'
    }
  },
  fs: { numberOfFiles: 15 },
  debug: [
    '2020-01-08T12:36:51.669Z config Detected Jest.\n',
    '2020-01-08T12:36:51.670Z config Configured Jest.\n',
    '2020-01-08T12:36:51.673Z project Wallaby Node version: v12.11.1\n',
    '2020-01-08T12:36:51.673Z project Wallaby config: F:\\github\\angular-testing-examples\\auto.detect\n',
    '2020-01-08T12:36:52.612Z project File cache: C:\\Users\\.WebStorm2019.3\\system\\wallaby\\projects\\8184444b4afb5aaa\n',
    '2020-01-08T12:36:52.666Z uiService Listening port 51235\n',
    '2020-01-08T12:36:52.690Z workers Parallelism for initial run: 2, for regular run: 2\n',
    '2020-01-08T12:36:52.690Z workers Starting run worker instance #0\n',
    '2020-01-08T12:36:52.690Z workers Starting run worker instance #1\n',
    '2020-01-08T12:36:52.691Z workers Web server is listening at 4221\n',
    '2020-01-08T12:36:52.713Z project Stopping process pool\n',
    '2020-01-08T12:36:52.714Z project File cache is up-to-date, starting full test run\n',
    '2020-01-08T12:36:52.716Z project Test run started; run priority: 3\n',
    '2020-01-08T12:36:52.717Z project Running all tests\n',
    '2020-01-08T12:36:52.720Z workers Starting test run, priority: 3\n',
    '2020-01-08T12:36:52.720Z nodeRunner Starting sandbox [worker #0, session #qcvnd]\n',
    '2020-01-08T12:36:52.720Z nodeRunner Preparing sandbox [worker #0, session #qcvnd]\n',
    '2020-01-08T12:36:53.083Z workers Started run worker instance (delayed) #0\n',
    '2020-01-08T12:36:53.084Z nodeRunner Prepared sandbox [worker #0, session #qcvnd]\n',
    '2020-01-08T12:36:53.084Z workers [worker #0, session #qcvnd] Running tests in sandbox\n',
    '2020-01-08T12:36:53.087Z workers Started run worker instance (delayed) #1\n',
    '2020-01-08T12:36:57.427Z workers Sandbox (active) [qcvnd] error: TypeError: C:\\Users\\.WebStorm2019.3\\system\\wallaby\\wallaby\\runners\\node\\jest@24.8.0\\jasmineInitializer.js: Emit skipped\n',
    '2020-01-08T12:36:57.434Z workers [qcvnd] Run 0 test(s), skipped 0 test(s)\n',
    '2020-01-08T12:36:57.437Z workers [qcvnd] Sandbox is responsive, closing it\n',
    '2020-01-08T12:36:57.438Z workers Failed to map the stack to user code, entry message: TypeError: C:\\Users\\.WebStorm2019.3\\system\\wallaby\\wallaby\\runners\\node\\jest@24.8.0\\jasmineInitializer.js: Emit skipped, stack: TypeError: C:\\Users\\.WebStorm2019.3\\system\\wallaby\\wallaby\\runners\\node\\jest@24.8.0\\jasmineInitializer.js: Emit skipped\n' +
      '    at getOutput (F:\\github\\angular-testing-examples\\node_modules\\ts-jest\\dist\\compiler.js:162:23)\n' +
      '    at Object.compile (F:\\github\\angular-testing-examples\\node_modules\\ts-jest\\dist\\compiler.js:208:25)\n' +
      '    at TsJestTransformer.process (F:\\github\\angular-testing-examples\\node_modules\\ts-jest\\dist\\ts-jest-transformer.js:101:41)\n' +
      '    at ScriptTransformer.transformSource (F:\\github\\angular-testing-examples\\node_modules\\@jest\\transform\\build\\ScriptTransformer.js:453:35)\n' +
      '    at ScriptTransformer._transformAndBuildScript (F:\\github\\angular-testing-examples\\node_modules\\@jest\\transform\\build\\ScriptTransformer.js:523:40)\n' +
      '    at ScriptTransformer.transform (F:\\github\\angular-testing-examples\\node_modules\\@jest\\transform\\build\\ScriptTransformer.js:579:25)\n' +
      '    at Runtime._execModule (F:\\github\\angular-testing-examples\\node_modules\\jest-runtime\\build\\index.js:842:53)\n' +
      '    at Runtime._loadModul\n',
    '2020-01-08T12:36:57.439Z project Test run finished\n',
    '2020-01-08T12:36:57.440Z project Processed console.log entries\n',
    '2020-01-08T12:36:57.440Z project Processed loading sequences\n',
    '2020-01-08T12:36:57.440Z project Processed executed tests\n',
    '2020-01-08T12:36:57.440Z project Processed code coverage\n',
    '2020-01-08T12:36:57.443Z project Test run result processed and sent to IDE\n'
  ]
}
```

_Originally posted by @felikf in https://github.com/wallabyjs/public/issues/2246#issuecomment-572030772_

**Comment by smcenlly** [comment]
Thanks for reporting the problem and for your Wallaby diagnostic report. We reproduced your issue, identifying a problem running jest automatic configuration on Windows machines when your project is on a different drive to the wallaby extension. This is fixed in Wallaby core v1.0.810+. If Wallaby does not update automatically for you, you may update to the latest version by closing all instances of WebStorm and re-opening WebStorm and starting Wallaby. Wallaby may use the last installed version the first time it runs, if so, please restart your run configuration after the update completes.

For reference, we have also created a clean Angular repo that uses jest with the [@briebug/jest-schematic](https://github.com/briebug/jest-schematic).

**Comment by felikf** [comment]
@smcenlly Thank you
